lms/templates/help_modal.html

```diff
@@ -8,7 +8,7 @@
 from django.core.urlresolvers import reverse
 from xmodule.tabs import CourseTabList
 from microsite_configuration import microsite
-platform_name = microsite.get_value("platform_name", settings.PLATFORM_NAME)
```

Comment:
@ziafazal do you mean that platform_name is not updating with this line of code ?